src/main/resources/application-dev.yaml

```diff
@@ -31,12 +32,13 @@ rolle:
 
 
 
-FAMILIE_INTEGRASJONER_URL: http://familie-integrasjoner-q1.teamfamilie
-SECURITYTOKENSERVICE_URL: https://sts-q1.preprod.local/SecurityTokenServiceProvider/
+FAMILIE_INTEGRASJONER_URL: https://familie-integrasjoner-q1.dev-fss-pub.nais.io
+SECURITYTOKENSERVICE_URL: https://security-token-service.dev.adeo.no/SecurityTokenServiceProvider/
 
 AZUREAD_DISCOVERY_URL: https://login.microsoftonline.com/navq.onmicrosoft.com/v2.0/.well-known/openid-configuration
 AZUREAD_TOKEN_ENDPOINT_URL: https://login.microsoftonline.com/navq.onmicrosoft.com/oauth2/v2.0/token
-PDL_URL: http://pdl-api-q1.pdl
+STS_URL: https://security-token-service.dev.adeo.no/security-token-service/rest/v1/sts/token?grant_type=client_credentials&scope=openid
```

Comment:
Tror ikke `security-token-service.dev.adeo.no` er tilgjengelig fra GCP? 
Hvis man virkelig skal bruke STS så må man bruke gateway hva jeg vet, men anbefaler å ta i bruk azuread i stedet, min erfaring var att det går ekstra treig med pdl+sts fra GCP

Reply:
ja, mot pdl skal vi straks gå for azureAD, mot soap tjenestene til økonomi derimot er det værre ;)

Reply:
Kan være greit å vurdere å legge inn det i familie-integrasjoner :)